app/questionnaire/placeholder_transforms.py

```diff
@@ -125,6 +125,31 @@ def parse_date(date):
                 date, PlaceholderTransforms.input_date_format_month_year_only
             ).replace(tzinfo=tzutc())
 
+    @staticmethod
+    def add(lhs, rhs):
+        if any([isinstance(lhs, str), isinstance(rhs, str)]):
+            return int(lhs) + int(rhs)
```

Comment:
This shouldn't be necessary. If the value is set in the schema it will be a number, and the answer store should be returning a number.

I've debugged this and I think it's because the answer store returns escaped answers wrapped in a `Markup` class. This is a bug in the placeholder rendering - the values should be escaped at the point they are interpolated into the string, rather than on the way out of the answer store. I think this bug will take some effort to investigate and fix, so I'm happy for it to be raised as an issue and fixed outside of this pull request.

Reply:
I created a separate card for this issue, do we keep this transform as it is?